forgettable-frontend/src/Components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import './SearchBar.css';
+import SearchIcon from '@material-ui/icons/Search';
+import CloseIcon from '@material-ui/icons/Close';
```

Comment:
If we import from '@mui/icons-material/Close' etc we can avoid having to install material-ui. (its confusing, but material-ui != mui lmaoo)

Reply:
o wat , I thought mui and material ui were the same lmao